### ISSUE TITLE
Datatypes for nodes table

### ISSUE THREAD

**Issue by mooreryan** [sherrillmix/taxonomizr#70]
I was going through the taxonomizr code and I noticed that while the taxonomy IDs are generally treated as integers (eg in the `accessionTaxa.taxa` column, `names.id` column, and `nodes.id` column), in the `nodes` table, the `parent` column uses `as.numeric` ([here](https://github.com/sherrillmix/taxonomizr/blob/5df3783cbbf3d9866834114c4ccb812a639beb54/R/taxa.R#L181)) and the resulting sqlite datatype becomes `REAL` rather than `INTEGER` (as observed when I dump the schema).

I'm wondering if `REAL` was chosen for that datatype rather than `INTEGER` for a specific reason?  

I haven't checked to see if it makes a performance difference on a join or not, and in fact I'm not even sure if sqlite would do any conversion between REAL and INTEGER before comparing (see 4.2. Type Conversions Prior To Comparison in https://www.sqlite.org/datatype3.html).

Anyway, I'm curious about your thoughts on it.

**Comment by sherrillmix** [comment]
Good catch. I don't think any special reason to be REAL and it just slipped by. The code has been working so some conversion must be going on in the background hence why this wasn't noticed. For example, sqlite does seem to compare INT and REAL without erroring out e.g.:

```
CREATE TABLE A (a int, b real);  
INSERT INTO A(a,b) VALUES (4,4);
INSERT INTO A(a,b) VALUES (4,4.0);
INSERT INTO A(a,b) VALUES (4,3.999999);
INSERT INTO A(a,b) VALUES (4,4.000001);
SELECT * FROM A WHERE a=b;
```

returning:
```
4|4.0
4|4.0
```

Might as well fix it since it seems an easy change. Should be updated on github now and I'll push to CRAN after the holidays. Thanks!

**Comment by sherrillmix** [comment]
The package should be on its way to CRAN now. Thanks!
